hangman/interface.py

```diff
@@ -71,32 +71,30 @@ def write_hint(self, hint, theme):
 
     def write_health(self, health):
         print("\nRemaining lives: {0}".format(health))
-        print("Remaining lives: {0}".format(health))
 
     def write_guessed_letters(self, guesses):
-        print("Letters guessed: {0}".format(guesses))
+        print("Letters guessed: {0}".format(guesses)) if len(guesses) > 0 else print("Letters guessed: [ ]")
```

Comment:
don't think you need this change anymore since `player.guesses` was changed back to a list.

Reply:
Reminder that this still needs to be removed!